*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 12:55:02 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 127.0.0.1:9090 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:44:08)  [GCC 7.3.0]
Python main interpreter initialized at 0x234c5c0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
ImportError: No module named 'three_server.three_server'
unable to load app 0 (mountpoint='') (callable not found or import error)
[DEBUG][2019-10-22 12:55:03,194][selector_events.py:65]Using selector: EpollSelector
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x234c5c0 pid: 1840 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 1840)
spawned uWSGI worker 1 (pid: 1845, cores: 1)
spawned uWSGI worker 2 (pid: 1846, cores: 1)
spawned uWSGI worker 3 (pid: 1847, cores: 1)
spawned uWSGI worker 4 (pid: 1848, cores: 1)
spawned uWSGI worker 5 (pid: 1849, cores: 1)
...brutally killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
binary reloading uWSGI...
chdir() to /shiw/three_server/script
closing all non-uwsgi socket fds > 2 (max_fd = 65535)...
found fd 3 mapped to socket 0 (127.0.0.1:9090)
running /usr/anaconda3/envs/three_server/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
[uwsgi-static] added mapping for /static => //shiw/three_server/front/dist/static
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 12:55:39 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 inherited INET address 127.0.0.1:9090 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:44:08)  [GCC 7.3.0]
Python main interpreter initialized at 0x2448150
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
ImportError: No module named 'three_server.three_server'
unable to load app 0 (mountpoint='') (callable not found or import error)
[DEBUG][2019-10-22 12:55:39,548][selector_events.py:65]Using selector: EpollSelector
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x2448150 pid: 1840 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 1840)
spawned uWSGI worker 1 (pid: 1865, cores: 1)
spawned uWSGI worker 2 (pid: 1866, cores: 1)
spawned uWSGI worker 3 (pid: 1867, cores: 1)
spawned uWSGI worker 4 (pid: 1868, cores: 1)
spawned uWSGI worker 5 (pid: 1869, cores: 1)
...brutally killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
binary reloading uWSGI...
chdir() to /shiw/three_server/script
closing all non-uwsgi socket fds > 2 (max_fd = 65535)...
found fd 3 mapped to socket 0 (127.0.0.1:9090)
running /usr/anaconda3/envs/three_server/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
[uwsgi-static] added mapping for /static => //shiw/three_server/front/dist/static
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 12:55:47 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 inherited INET address 127.0.0.1:9090 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:44:08)  [GCC 7.3.0]
Python main interpreter initialized at 0x25e2150
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
ImportError: No module named 'three_server.three_server'
unable to load app 0 (mountpoint='') (callable not found or import error)
[DEBUG][2019-10-22 12:55:48,330][selector_events.py:65]Using selector: EpollSelector
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x25e2150 pid: 1840 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 1840)
spawned uWSGI worker 1 (pid: 1877, cores: 1)
spawned uWSGI worker 2 (pid: 1878, cores: 1)
spawned uWSGI worker 3 (pid: 1879, cores: 1)
spawned uWSGI worker 4 (pid: 1880, cores: 1)
spawned uWSGI worker 5 (pid: 1881, cores: 1)
Tue Oct 22 12:55:53 2019 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:55:53 2019 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:55:53 2019 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:55:53 2019 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:55:53 2019 - uWSGI worker 5 screams: UAAAAAAH my master disconnected: i will kill myself !!!
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 12:57:41 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 127.0.0.1:9090 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:44:08)  [GCC 7.3.0]
Python main interpreter initialized at 0x18d65c0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
ImportError: No module named 'three_server.three_server'
unable to load app 0 (mountpoint='') (callable not found or import error)
[DEBUG][2019-10-22 12:57:41,665][selector_events.py:65]Using selector: EpollSelector
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x18d65c0 pid: 2243 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 2243)
spawned uWSGI worker 1 (pid: 2248, cores: 1)
spawned uWSGI worker 2 (pid: 2249, cores: 1)
spawned uWSGI worker 3 (pid: 2250, cores: 1)
spawned uWSGI worker 4 (pid: 2251, cores: 1)
spawned uWSGI worker 5 (pid: 2252, cores: 1)
Tue Oct 22 12:58:10 2019 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:58:10 2019 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:58:10 2019 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:58:10 2019 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 12:58:10 2019 - uWSGI worker 5 screams: UAAAAAAH my master disconnected: i will kill myself !!!
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:01:30 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:05:17 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 127.0.0.1:9090 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:44:08)  [GCC 7.3.0]
Python main interpreter initialized at 0x24ed5c0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
ImportError: No module named 'three_server.three_server'
unable to load app 0 (mountpoint='') (callable not found or import error)
[DEBUG][2019-10-22 13:05:17,783][selector_events.py:65]Using selector: EpollSelector
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x24ed5c0 pid: 3413 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3413)
spawned uWSGI worker 1 (pid: 3418, cores: 1)
spawned uWSGI worker 2 (pid: 3419, cores: 1)
spawned uWSGI worker 3 (pid: 3420, cores: 1)
spawned uWSGI worker 4 (pid: 3421, cores: 1)
spawned uWSGI worker 5 (pid: 3422, cores: 1)
[pid: 3418|app: 0|req: 1/1] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:05:21 2019] GET / => generated 5393 bytes in 69 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3419|app: 0|req: 1/2] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:05:22 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 78 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
[pid: 3420|app: 0|req: 1/3] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:05:40 2019] GET / => generated 5393 bytes in 72 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3421|app: 0|req: 1/4] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:05:40 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 77 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
[pid: 3422|app: 0|req: 1/5] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:08:25 2019] GET / => generated 5393 bytes in 69 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3418|app: 0|req: 2/6] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:08:25 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 19 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
[pid: 3419|app: 0|req: 2/7] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:08:27 2019] GET / => generated 5393 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3420|app: 0|req: 2/8] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:08:27 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 19 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
[pid: 3421|app: 0|req: 2/9] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:08:27 2019] GET / => generated 5393 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3422|app: 0|req: 2/10] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:08:27 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 19 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
[pid: 3418|app: 0|req: 3/11] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:08:28 2019] GET / => generated 5393 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3419|app: 0|req: 3/12] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:08:28 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
[pid: 3420|app: 0|req: 3/13] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:08:28 2019] GET / => generated 5393 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3421|app: 0|req: 3/14] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:08:28 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
[pid: 3422|app: 0|req: 3/15] 203.177.117.4 () {44 vars in 1123 bytes} [Tue Oct 22 13:08:29 2019] GET / => generated 5393 bytes in 1 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 3418|app: 0|req: 4/16] 203.177.117.4 () {44 vars in 1269 bytes} [Tue Oct 22 13:08:29 2019] GET /api/system/user/get_user_info => generated 2934 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 130 bytes (1 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:03 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 127.0.0.1:9090 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:44:08)  [GCC 7.3.0]
Python main interpreter initialized at 0x1225450
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
ImportError: No module named 'three_server.three_server'
unable to load app 0 (mountpoint='') (callable not found or import error)
[DEBUG][2019-10-22 13:15:03,528][selector_events.py:65]Using selector: EpollSelector
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1225450 pid: 3678 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 3678)
spawned uWSGI worker 1 (pid: 3682, cores: 1)
spawned uWSGI worker 2 (pid: 3684, cores: 1)
spawned uWSGI worker 3 (pid: 3685, cores: 1)
spawned uWSGI worker 4 (pid: 3686, cores: 1)
spawned uWSGI worker 5 (pid: 3687, cores: 1)
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:04 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:06 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:10 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:20 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:21 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:23 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Oct 22 13:15:26 2019] ***
compiled with version: 4.8.2 20140120 (Red Hat 4.8.2-15) on 15 August 2018 17:45:58
os: Linux-4.4.0-151-generic #178-Ubuntu SMP Tue Jun 11 08:30:22 UTC 2019
nodename: iZj6cgqdhf0dbc2j2gebpyZ
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /shiw/three_server/script
writing pidfile to /shiw/three_server/script/uwsgi.pid
detected binary path: /usr/anaconda3/envs/three_server/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /shiw/three_server/
your processes number limit is 64032
your memory page size is 4096 bytes
detected max file descriptor number: 65535
building mime-types dictionary from file /etc/mime.types...552 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (127.0.0.1:9090).
bind(): Address already in use [core/socket.c line 769]
Tue Oct 22 13:19:27 2019 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 13:19:27 2019 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 13:19:27 2019 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 13:19:27 2019 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Tue Oct 22 13:19:27 2019 - uWSGI worker 5 screams: UAAAAAAH my master disconnected: i will kill myself !!!
